azurepipelines/e2e_test/scenarios/ubuntu-18.04-amd64/vm_setup/devicesetup.py

```diff
@@ -63,13 +63,34 @@ def main():
                     }
 
     #
-    # Write the configuration out to disk so we can install it as a part of the test
+    # Write the DU configuration out to disk so we can install it as a part of the test
     #
 
     with open('du-config.json','w') as jsonFile:
         configJson = json.dumps(duConfigJson)
         jsonFile.write(configJson)
 
+    #
+    # Create the config.toml contents
+    #
+    configTomlInfo = """\
+[provisioning]
+source = "manual"
+connection_string = "{connectionString}"
+
+[aziot_keys]
+
+[preloaded_keys]
+
+[cert_issuance]
+
+[preloaded_certs]
+
+""".format(connectionString=connectionString)
+
+    with open('config.toml','w') as configToml:
```

Comment:
Just curious, do we delete this file later? If not, we should.

Reply:
we don't need to. The system will handle the cleanup since we're on a 1:1 provisioning scenario to VM run.

@shiyi-peng  is working on a rework that will move this to vm-runner (1 producer: many consumer model) which will require some cleanup.